apps/web/components/SearchInput/SearchInput.tsx

```diff
@@ -427,24 +429,32 @@ const Results: FC<{
               {(search.results.items as Article[] &
                 LifeEventPage[] &
                 AboutPage[] &
-                News[])
+                News[] &
+                SubArticle[])
                 .slice(0, 5)
-                .map(({ id, title, slug, __typename }) => {
+                .map((item) => {
                   const { onClick, ...itemProps } = getItemProps({
                     item: '',
                   })
                   return (
                     <div
-                      key={id}
+                      key={item.id}
                       {...itemProps}
                       onClick={(e) => {
                         onClick(e)
                         onRouting()
                       }}
                     >
-                      <Link {...linkResolver(__typename as LinkType, [slug])}>
+                      <Link
+                        {...linkResolver(
+                          item.__typename as LinkType,
+                          item.slug.split('/').length > 1
+                            ? item.slug.split('/')
+                            : [item.slug],
```

Comment:
Missed this earlier, same as below:
```suggestion
                          item.slug.split('/')
```

Reply:
I should've known as well